src/core/expences/controllers/expenses.controller.ts

```diff
@@ -34,8 +35,13 @@ export class ExpensesController {
     description: 'Internal server error',
   })
   @Get()
-  findAll() {
-    return this.expensesService.findAll();
+  findAll(@Query('userId') userId: string) {
```

Comment:
```suggestion
  @Get('/byUserId/:userId')
```
better use params then query. It can help minimize injections